docs/pages/guides/configuring-statusbar.mdx

```diff
@@ -123,6 +138,7 @@ Example:
 }
 ```
 
-### Place an empty `View` on top of your screen
+### Workaround #2
```

Comment:
The title here doesn't tell me much when I'm going through it. I'd still like the section headers to be descriptive. I do get the point of having short header titles.

My suggestion here would be to use something like: "Empty `View`"